PFR/entropy_basic.lean

```diff
@@ -223,7 +223,7 @@ lemma measureEntropy_le_log_card [MeasurableSingletonClass S] (μ : Measure S) :
       rw [Fintype.card_eq_zero]
       simp
     | inr h =>
-      refine log_nonneg ?_
+      apply log_nonneg
```

Comment:
This is a regression IMO. It's hard to know what `apply` does and that has unexpected behavior.

Reply:
I've been using `apply` this way since the first day I came to Lean and I find it very idiomatic, but I'll follow your advice and stop pushing this.